src/smt/set_defaults.cpp

```diff
@@ -335,6 +358,15 @@ void SetDefaults::setDefaultsPre(Options& opts)
                            sygusInference,
                            options::SygusInferenceMode::OFF,
                            "internal subsolver");
+    // proofs/unsat cores not yet supported with CaDiCaL
```

Comment:
I would expect this to be unecessary now with the other changes.

Reply:
indeed

Reply:
SG, can you delete the commented out code now?